### DISCUSSION TITLE
【来整点小说？】普通人的故事

### DISCUSSION THREAD

**Discussion by Chilin987** [share121/inter-knot#3298] [其他]
就  随便写写脑洞
哪天闲着就会掉落一点 ~~头发~~ 更新
不是严谨的设定党所以可能有谬误，欢迎指正√<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

### **REMIND!!!**
<ins>和游戏正式剧情无关，只采用部分设定和世界观</ins>
<ins>自机角色（抽卡得到的代理人） ~~（应该？）~~ 不会出现</ins>
如有雷同.....?不知道啦反正我还没读过绝的同人作品你说是就是吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

---
   
1.
林军林云父子俩是从「艾利都」空洞陷落中侥幸逃脱的。
空洞中建筑交叠，错乱的空间中，钢筋水泥混杂着玻璃碎片自空中随机落下。眼看一个新的缝隙在林云头顶生成，林军赶紧扑过去用身体保护儿子。他的双腿被坠落的杂物碾压，剧痛使他瞬间失去了意识。
      
2.
幸而另一个突然出现的裂隙将他们送到了空洞边缘。林云拖着父亲的身体赶到空洞外的临时救助站时，他的双腿已经接近坏死，只能截肢保命。
事发突然，两人都未能带上什么财物，林军变卖了自己的金表补贴家用，林云则开始四处游荡打听有没有地方招临时工。
      
3.
林云找到了一个杂货店搬运工的活，店铺的二楼住着老板一家。老板听说他有个瘸腿的父亲，给他找了个附近的平房租住。林云勤勤恳恳干活，林军则在家中养病，闲事找邻居学了一些编织手艺，日子还能过得下去。
      
4.
然而好景不长，艾利都旧有的势力经空洞后实力大减，原本周边被压制的帮派势力开始蠢蠢欲动。冲突中，林云之前打零工的店受到了波及。店铺主人抛弃了这里，告诉林云他将会逃往更安全的地界，店里的杂物都会留下。林云挑选了一些食物和生活必需品带回家。帮林军换完截肢处的敷料后，他做出了重返空洞边缘拾荒的决定。
      
5.
于是，失去工作的第一天，他背着双肩包与往常一样出了门。祈祷着在家的林军不会遇到麻烦，祈祷着自己能有大量收获，祈祷着空洞不会突然扩张。

**Comment by yzysrewrwqr** [comment]
为啥要加序号？

**Reply by Chilin987** [reply]
最开始是按顺序记录的，后来就懒得删啦（

**Comment by Chilin987** [comment]
他的运气很好，与逃离空洞那时相比，空洞并没有扩张太多，甚至缩小了一点。他搜刮了一圈，找到了大堆的各种废弃金属和小型首饰。他将背包塞满，剩余的金属全部埋藏进就近的一个小地下室里。顺利回到聚居点之后，他将背包里的物件换成食物，带回家中。
      
这次的收获虽然不多，但与之前打工的薪水相比还是高出不少的，他盘算着藏起来的金属数量，应该能换到支撑一个礼拜的食物。但这之后应该怎么办？他不知道。他看着林军递过来的半个土豆，挤出了一个苦涩的笑容。
      
林云背着几个麻袋出门，推着一推车的金属回到市场交易，换到的食物又背回家中。
或许是最近来交易的难民太多了，灰头土脸的他和他的大量金属才没有引起什么人的注意。他这么认为着。
      
又一次的，他打算运走最后一批废铜烂铁。然而面前出现的一群人打断了他的动作，那是附近的帮派。他识趣的退走，留下了所有物资。
周边的一切都已经被搜刮干净了。前所未有的灾难把难民们逼成了一群蝗虫，想要活下去，或许只有铤而走险了。
      
墙上，一张广告摇摇欲坠，赛棱生物正在计划去空洞中找回灾难中走失的董事长独女。他们招聘的打手，工资待遇相当不错，而且进空洞前一次结清。
而进空洞，肯定要求有以太适性。
他握紧了手上的试剂盒。

**Comment by Chilin987** [comment]
Day2.1

黄色。
命运似乎还是站在他这边的。
      
不管怎么说，他拥有参加的资格了。哪怕只是参加一次，也足够他和父亲吃上两三个月的饱饭。他捧着试剂盒，快乐溢满了眉间。
      
在他的对面，林军斜靠在床上捻麻绳。他最开始捻的歪七扭八，后来请教偶尔会给他送饭的邻居，又耐着性子反复练习，才有人愿意收。一天搓三十米长的细麻绳，只是刚刚能换来自己的饭钱。
      
他也曾为自己的腿惋惜，也想过要不要干脆自尽把希望和未来留给林云，不过人总是更想活着的，而且，一切都在好起来不是吗？他不太明白儿子为什么今天这么开心，但是只要他快乐，自己也就很欢喜了。于是，他又将手上的麻绳接续了下去，吱呀吱呀的绕在线轮上。
      
和所有其他参加的打手一样，林云预支了这次的工资。他把林军安置到一个更安全的地方，并联系医生给他按上了假肢。虽然很不方便，还需要每三个月更换一次，但至少他能缓慢地走动了。
      
第一次的搜查其实更像是探路。他们只在表层的空洞进行了探索，领队指哪儿去哪儿。

听人说，这次带队的老哥经常进出空洞，心里有一张地图，哪儿能走哪儿不能，都心里有数。林云也不知道他们说的是真是假，只是一味跟着。
      
遇到落单的以骸他就跟着一群人冲上去，遇到大片的——喔，好像进来这大半天都没有遇到那种大场面，听说上一个目击以骸潮的家伙是抛弃了自己的所有同伴才逃出来，后来不知道为啥精神崩溃了，竟又疯疯癫癫地又跑回了空洞。
